lib/App.server.js

```diff
@@ -262,9 +262,13 @@ App.prototype._watchViews = function(filenames, filename, namespace) {
   var watcher = chokidar.watch(filenames);
   watcher.on('change', function() {
     watcher.close();
-    app.loadViews(filename, namespace);
-    app._updateScriptViews();
-    app._refreshClients();
+    try {
```

Comment:
Which functions/line do the errors we're trying to capture come from? Can we get away with having less functions within the try block?

The way the current code is setup is that we close the `watcher` on every change event. The watcher is then recreated within `loadViews()` and `loadStyles()`. This means that if there was an error in one of these functions before the next watcher is created then the app would cease to live reload

Perhaps this isn't an issue if the errors we're trying to handle always appear after `loadViews()` and `loadStyles()`. If that's the case, then we should consider removing those calls from within the `try` block